app/src/androidTest/java/com/appunite/loudius/util/MockWebServerRule.kt

```diff
@@ -0,0 +1,93 @@
+package com.appunite.loudius.util
+
+import android.util.Log
+import com.appunite.loudius.di.TestInterceptor
+import io.mockk.MockKMatcherScope
+import io.mockk.every
+import io.mockk.mockk
+import okhttp3.HttpUrl
+import okhttp3.Interceptor
+import okhttp3.Response
+import okhttp3.mockwebserver.Dispatcher
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okhttp3.mockwebserver.RecordedRequest
+import okhttp3.mockwebserver.SocketPolicy
+import org.intellij.lang.annotations.Language
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import strikt.api.Assertion
+import strikt.api.expectThat
+import strikt.assertions.isNotNull
+
+private const val TAG = "MockWebServerRule"
+
+class MockWebServerRule : TestRule {
+
+    val dispatcher: Dispatcher = mockk {
+        every { shutdown() } returns Unit
+        every { peek() } returns MockResponse().apply { this.socketPolicy = SocketPolicy.KEEP_OPEN }
+        every { dispatch(any()) } answers {
+            val request = it.invocation.args[0] as RecordedRequest
+            Log.w(TAG, "Request is not mocked: ${request.method} ${request.path}")
```

Comment:
suggestion/out-of-scope: I found some tool that I've written before for generating Curls for given requests: https://github.com/jacek-marchwicki/blog/pull/2/files#diff-a3868ac818c91fa781d8971ef674100cb8902486617f216532ff273888289851R109-R152. This might be useful to call real method.